dev/export-env.sh

```diff
@@ -0,0 +1,29 @@
+# Export environment variables from the .env file in the first argument.
```

Comment:
This file + the rest of the files in this directory are based on https://github.com/girder/cookiecutter-girder-4/tree/master/%7B%7B%20cookiecutter.project_slug%20%7D%7D/dev